### PR TITLE
Pass --copt, --cxxopt, --conlyopt, --linkopt to cmake_external/config…

### PR DESCRIPTION
**Pull request by irengrig**
…ure_make rules

- explicitly pass the values of these options to corresponding compilation/link flags lists; add them to the end of the lists of they are not already there
- please see the test in test/standard_cxx_flags_test

### CODE REVIEW

**Comment by irengrig** [comment]
Fixes #154

File: test/standard_cxx_flags_test/tests.bzl [hunk 1]
```diff
@@ -0,0 +1,44 @@
+""" TODO """
```
Comment (hlopko) [inline test/standard_cxx_flags_test/tests.bzl]: :)

File: tools/build_defs/cc_toolchain_util.bzl [hunk 2]
```diff
@@ -264,39 +264,40 @@ def get_flags_info(ctx):
     Args:
         ctx - rule context
     """
-    cc_toolchain = find_cpp_toolchain(ctx)
+    cc_toolchain_ = find_cpp_toolchain(ctx)
```
Comment (hlopko) [inline tools/build_defs/cc_toolchain_util.bzl]: Why trailing underscore?

File: tools/build_defs/cc_toolchain_util.bzl [hunk 3]
```diff
@@ -264,39 +264,40 @@ def get_flags_info(ctx):
...
     feature_configuration = cc_common.configure_features(
-        cc_toolchain = cc_toolchain,
+        cc_toolchain = cc_toolchain_,
         requested_features = ctx.features,
         unsupported_features = ctx.disabled_features,
     )
-    copts = ctx.attr.copts if hasattr(ctx.attr, "copts") else []
 
-    return CxxFlagsInfo(
+    copts = (ctx.fragments.cpp.copts + ctx.fragments.cpp.conlyopts) or []
```
Comment (hlopko) [inline tools/build_defs/cc_toolchain_util.bzl]: Why or []? ctx.fragments.cpp.*opts always returns empty collection, doesn't it?

**Pull request merged**